crates/typst-library/src/math/ctx.rs

```diff
@@ -204,9 +205,11 @@ impl<'a, 'b, 'v> MathContext<'a, 'b, 'v> {
             }
             let text: EcoString = text.chars().map(|c| style.styled_char(c)).collect();
             let text = TextElem::packed(text)
-                .styled(TextElem::set_top_edge(TextEdge::Bbox(BboxVertical::BboxTop)))
-                .styled(TextElem::set_bottom_edge(TextEdge::Bbox(
-                    BboxVertical::BboxBottom,
+                .styled(TextElem::set_top_edge(TextTopEdge::Metric(
+                    TopEdgeMetric::BoundingBox,
+                )))
+                .styled(TextElem::set_bottom_edge(TextBottomEdge::Metric(
+                    BottomEdgeMetric::BoundingBox,
```

Comment:
I'm not sure what the right way is to make a new `TextElem` and set style properties.  I'm pretty sure that what I'm doing here is not the recommended route.  I had tried
```rust
        let text = TextElem::new(text)
            .spanned(span)
            .with_top_edge(TextTopEdge::Metric(TopEdgeMetric::BoundingBox))
            .with_bottom_edge(TextBottomEdge::Metric(BottomEdgeMetric::BoundingBox))
            .pack();
```
but the edge settings didn't have any effect.  Ideally, one wants to have a default of bounding box edges for text in math mode, but also allow these default settings to be overridden via the style mechanism. But I haven't wrapped my head around the architecture yet and could use a hand here.

Reply:
You've probably hit a weird edges case of how text elements work (they do not typically have inherent fields which `.with_` creates). The way you've done it is okay.